azurerm/tags.go

```diff
@@ -79,16 +84,24 @@ func expandTags(tagsMap map[string]interface{}) map[string]*string {
 	return output
 }
 
-func flattenAndSetTags(d *schema.ResourceData, tagsMap map[string]*string) {
+func flattenAndSetTags(d *schema.ResourceData, tagsMap map[string]*string, skipPropNames ...string) {
 	if tagsMap == nil {
 		d.Set("tags", make(map[string]interface{}))
 		return
 	}
 
 	output := make(map[string]interface{}, len(tagsMap))
 
+	// Only first optional parameter will be processed.
```

Comment:
What about the rest of the parameters?

Reply:
This is going to support optional parameter. A better way is to build a map instead. Let me refactor the code.

Reply:
let's instead revert the changes to this method entirely and leave it in the single resource where this change is needed - this method's already doing too much